flasksrc/quiz.py

```diff
@@ -0,0 +1,43 @@
+from flask import Flask, request
+from flask_restful import Resource, Api, abort
+
+from flasksrc.db import get_db
+
+class ToyNetQuiz(Resource):
+    def get(self, quiz_id):
+        db = get_db()
+        error = None
+
+        rows_question = db.execute(
```

Comment:
I forgot to wrap this in a `try` `catch` in the PR you copied:
https://github.com/Project-Reclass/toynet-flask/blob/main/flasksrc/value.py#L10

let's do that here and remove `error` on line 9.